docs/src/rules/class-methods-use-this.md

```diff
@@ -200,3 +202,126 @@ class A {
 ```
 
 :::
+
+### ignoreOverrideMethods
+
+```js
+"class-methods-use-this": [<enabled>, { "ignoreOverrideMethods": true | false }]
+```
+
+The `ignoreOverrideMethods` option ignores members that are marked with the `override` modifier. (default: `false`)
+
+Examples of **incorrect** TypeScript code for this rule with the `{ "ignoreOverrideMethods": false }` option (default):
+
+::: incorrect
+
+```ts
+/*eslint class-methods-use-this: ["error", { "ignoreOverrideMethods": false }] */
+
+class Derived extends Base {
+    override foo() {};
+}
+```
+
+:::
+
+Examples of **correct** TypeScript code for this rule with the `{ "ignoreOverrideMethods": false }` option (default):
+
+::: correct
+
+```ts
+/*eslint class-methods-use-this: ["error", { "ignoreOverrideMethods": false }] */
+
+class Derived extends Base {
+    override foo() {
+        this.bar = "Hello World";
+    };
+}
+```
+
+:::
+
+Examples of **correct** TypeScript code for this rule with the `{ "ignoreOverrideMethods": true }` option:
+
+::: correct
+
+```ts
+/*eslint class-methods-use-this: ["error", { "ignoreOverrideMethods": true }] */
+
+class Derived extends Base {
+    override foo() {};
+}
```

Comment:
As mentioned in the issue description, it should work similarly to the [corresponding ts rule](https://typescript-eslint.io/rules/class-methods-use-this/#ignoreoverridemethods)

```suggestion
abstract class Base {
  abstract method(): void;
  abstract property: () => void;
}

class Derived extends Base {
  override method() {}
  override property = () => {};
}
```

Reply:
I think @snitin315's suggestion is a good one. :+1: